clang/include/clang-c/Index.h

```diff
@@ -2145,7 +2145,11 @@ enum CXCursorKind {
    */
   CXCursor_OMPScopeDirective = 306,
 
-  CXCursor_LastStmt = CXCursor_OMPScopeDirective,
+  /** OpenACC Compute Construct.
+   */
+  CXCursor_OpenACCComputeConstruct = 307,
```

Comment:
I would suggest to save some constant number for future OpenMP nodes. How about start it from, say, 320 for OpenACC nodes? Just to avoid mixing of OpenMP and OpenACC nodes

Reply:
Thats a good idea, and I think 320 gives me plenty of room before we start getting into the 350 of 'translation unit'.  There are currently 20 'construct' types for OpenACC, though I am likely to merge a couple of them (as I am merging 3 different ones in this patch).  Patch incoming.